src/rebar_file_utils.erl

```diff
@@ -189,16 +189,28 @@ cp_r([], _Dest) ->
     ok;
 cp_r(Sources, Dest) ->
     case os:type() of
-        {unix, _} ->
+        {unix, Os} ->
             EscSources = [rebar_utils:escape_chars(Src) || Src <- Sources],
             SourceStr = rebar_string:join(EscSources, " "),
+            % On darwin the following cp command will cp everything inside
+            % target vs target and everything inside, so we chop the last char
+            % off if it is a '/'
+            Source = case {Os == darwin, lists:last(SourceStr) == 47} of
+                {true, true} ->
+                    string:sub_string(SourceStr, 1, length(SourceStr) - 1);
+                {true, false} ->
+                    SourceStr;
+                {false, _} ->
+                    SourceStr
+            end,
+            EscSources = [rebar_utils:escape_chars(Src) || Src <- Sources],
```

Comment:
`EscSources` line is accidentally doubled.

Reply:
Thank you for the catch! 👍